From: Kurt Hindenburg <kurt.hindenburg@gmail.com>
Date: Fri, 21 Apr 2017 21:40:32 -0400
Subject: Add check for valid container (possible crash fix)

Possible fix for crashes from ambiguous shortcuts.  I have been unable
to duplicate the crashes.

CCBUG: 375540
---
 src/ViewManager.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/ViewManager.cpp b/src/ViewManager.cpp
index c187e646..50d650d1 100644
--- a/src/ViewManager.cpp
+++ b/src/ViewManager.cpp
@@ -905,6 +905,9 @@ void ViewManager::saveSessions(KConfigGroup& group)
     // first: sessions in the active container, preserving the order
     ViewContainer* container = _viewSplitter->activeContainer();
     Q_ASSERT(container);
+    if (container == nullptr) {
+        return;
+    }
     TerminalDisplay* activeview = qobject_cast<TerminalDisplay*>(container->activeView());
 
     QListIterator<QWidget*> viewIter(container->views());
